packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1041,6 +1042,127 @@ curl -v -X POST \
 }
 ```
 
+#### Add Smart Card Identity Provider
+
+
+Adds a new Smart Card `X509` type IdP to your organization
+
+##### Request Example
+
+> **Notes:** You must first add the IdP's server certificate to the IdP key store before you can add a Smart Card `X509` IdP with a `kid` credential reference. You need to upload the whole trust chain as single key using the [Key Store API](#add-x-509-certificate-public-key).
+Depending on the information stored in the smart card select proper template "idpuser.subjectAltNameEmail" or "idpuser.subjectAltNameUpn".
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "X509",
+  "status": "ACTIVE",
+  "features": [],
+  "name": "Smart Card IDP Name",
+  "protocol": {
+    "type": "MTLS",
+    "credentials": {
+      "trust": {
+        "revocation": "CRL",
+        "revocationCacheLifetime": 2880,
+        "issuer": "your-issuer",
+        "kid": "your-kid"
```

Comment:
Where do we get the issuer information from? Does that value show up when the CA chain is uploaded to the key store using the key store API?

Reply:
The key store operations are described here - https://developer.okta.com/docs/reference/api/idps/#add-x-509-certificate-public-key.

When you create new key you will get part of the response "kid", which you can use to create your IdP configuration. Issuer is text description of the cert chain.

The key store operations when you retrieve the information for specific "kid" you will see the whole chain of certs.

Reply:
So the Issuer value can be anything and doesn't have to be in the format as it shows up on the UI such as ""CN=Test Smart Card, OU=Test OU, O=Test O, C=US"